appinfo/routes.php

```diff
@@ -339,5 +339,31 @@
 				'apiVersion' => 'v2(\.[1-3])?'
 			]
 		],
+		// Submissions linking with file in cloud
+		[
+			'name' => 'api#linkFile',
+			'url' => '/api/{apiVersion}/link/link/{fileFormat}',
```

Comment:
```suggestion
			'url' => '/api/{apiVersion}/form/link/{fileFormat}',
```

A little typo slipped in ;)